cartridge/webui/api-ddl.lua

```diff
@@ -39,81 +37,41 @@ local function graphql_get_schema()
             "Current instance isn't bootstrapped yet"
         )
     end
-    local schema_yml = confapplier.get_readonly(_section_name)
-    if schema_yml == nil or schema_yml == '' then
-        schema_yml = ddl_manager._example_schema
-    end
 
-    return {
-        as_yaml = schema_yml,
-    }
+    local ddl_manager = assert(service_registry.get('ddl-manager'))
+    return {as_yaml = ddl_manager.get_clusterwide_schema_yaml()}
 end
 
 local function graphql_set_schema(_, args)
     if confapplier.get_readonly() == nil then
-        return nil, GetSchemaError:new(
+        return nil, CheckSchemaError:new(
```

Comment:
I suggest adding a deprecation warning for schema-related methods

Reply:
As we discussed it orally, we'll decide a little bit later. In particular, completely remove it in scope of https://github.com/tarantool/cartridge/issues/1085#issuecomment-817936942